### PR TITLE
[UPD] Brainbean Apps => CorporateHub

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@pedrobaeza this one as well please

**Comment by alexey-pelykh** [comment]
Travis is red on linting, but that's not related to PR changes at all

**Comment by alexey-pelykh** [comment]
@OCA/social-maintainers would it be possible to take a look at this?

**Comment by pedrobaeza** [comment]
Sorry, but we can't merge something red.

**Comment by alexey-pelykh** [comment]
@pedrobaeza fixed by rebasing, as I've mentioned - that red was nothing to do with PR

**Comment by alexey-pelykh** [comment]
cc @OCA/social-maintainers

**Comment by alexey-pelykh** [comment]
@pedrobaeza if you don't mind :)

File: mail_activity_reminder/README.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Mail Activity Reminder
     :target: https://runbot.odoo-community.org/runbot/205/12.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```
Comment (pedrobaeza) [inline mail_activity_reminder/README.rst]: You have edited this manually and this shouldn't be touched

Comment (alexey-pelykh) [reply mail_activity_reminder/README.rst]: It makes no difference - the file will be regenerated anyhow

Comment (pedrobaeza) [reply mail_activity_reminder/README.rst]: Yes, but more noise for this simple change. Can you do it at once?

Comment (alexey-pelykh) [reply mail_activity_reminder/README.rst]: okay okay, you win as always :P, let me revert that specific file 👍

Comment (alexey-pelykh) [reply mail_activity_reminder/README.rst]: done

**Comment by pedrobaeza** [comment]
Hehe, I didn't mean that, as an extra commit will still land, but let's move on:

/ocabot merge patch

**Pull request merged**